### PR TITLE
cdc: add docs about the claim check.

### PR DESCRIPTION
**Pull request by 3AceShowHand**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-sink-to-kafka.md [hunk 1]
```diff
@@ -353,4 +353,74 @@ Kafka 消费者收到消息之后，首先检查 `onlyHandleKey` 字段。如果
 
 > **警告：**
 >
-> 在 Kafka 消费者处理数据并查询 TiDB 时，可能发生数据已经被 GC 的情况。你需要[调整 TiDB 集群的 GC Lifetime 设置](/system-variables.md#tidb_gc_life_time-从-v50-版本开始引入) 为一个较大的值，以避免该情况。
\ No newline at end of file
+> 在 Kafka 消费者处理数据并查询 TiDB 时，可能发生数据已经被 GC 的情况。你需要[调整 TiDB 集群的 GC Lifetime 设置](/system-variables.md#tidb_gc_life_time-从-v50-版本开始引入) 为一个较大的值，以避免该情况。
+
+### 发送大消息到外部存储
```
Comment (fubinzh) [inline ticdc/ticdc-sink-to-kafka.md:358]: 建议添加说明，CDC 不会清理外部存储服务上的消息，用户需要自己管理其生命周期

File: ticdc/ticdc-sink-to-kafka.md [hunk 2]
```diff
@@ -292,6 +292,29 @@ SELECT COUNT(*) FROM INFORMATION_SCHEMA.TIKV_REGION_STATUS WHERE DB_NAME="databa
 
 Kafka Topic 对可以接收的消息大小有限制，该限制由 [`max.message.bytes`](https://kafka.apache.org/documentation/#topicconfigs_max.message.bytes) 参数控制。当 TiCDC Kafka sink 在发送数据时，如果发现数据大小超过了该限制，会导致 changefeed 报错，无法继续同步数据。为了解决这个问题，TiCDC 新增一个参数 `large-message-handle-option` 并提供如下解决方案。
 
+目前，该功能支持 Canal-JSON 和 Open Protocol 两种编码协议。使用 Canal-JSON 协议时，你需要在 `sink-uri` 中指定 `enable-tidb-extension=true` 参数。
+
+### TiCDC 层数据压缩功能
+
+从 v7.4.0 开始，TiCDC Kafka sink 支持在编码消息后立马对数据进行压缩，然后和消息大小限制参数比较。该功能能够有效减少超过消息大小限制的情况发生。
```
Comment (ran-huang) [inline ticdc/ticdc-sink-to-kafka.md:299]:
```suggestion
从 v7.4.0 开始，TiCDC Kafka sink 支持在编码消息后立即对数据进行压缩，并与消息大小限制参数比较。该功能能够有效减少超过消息大小限制的情况发生。
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 3]
```diff
@@ -292,6 +292,29 @@ SELECT COUNT(*) FROM INFORMATION_SCHEMA.TIKV_REGION_STATUS WHERE DB_NAME="databa
...
+[sink.kafka-config.large-message-handle]
+# 该参数从 v7.4.0 开始引入
+# 默认为空，即不开启编码时的压缩功能
+# 可选值有 "none"、"lz4"、"snappy", 默认为 "none"
+large-message-handle-compression = "none"
+```
+
+该功能和 Kafka producer 的压缩功能不同：
+
+* `large-message-handle-compression` 中指定的压缩算法，它所启用的压缩功能是在单条 Kafka 消息上，并且是在和消息大小限制参数比较之前进行的。
```
Comment (ran-huang) [inline ticdc/ticdc-sink-to-kafka.md:313]:
```suggestion
* `large-message-handle-compression` 中指定的压缩算法，它启用的是对单条 Kafka 消息进行压缩，并且压缩是在与消息大小限制参数比较之前进行。
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 4]
```diff
@@ -292,6 +292,29 @@ SELECT COUNT(*) FROM INFORMATION_SCHEMA.TIKV_REGION_STATUS WHERE DB_NAME="databa
...
+* 用户可以在 `sink-uri` 中配置压缩算法，它所启用的压缩功能应用在整个发送数据请求，其中包含多条 Kafka 消息，并且是在和消息大小限制参数比较之后进行的。
+
+开启了 `large-message-handle-compression` 之后，消费者收到的消息是经过特定压缩协议编码的，消费者应用者需要使用指定的压缩协议进行数据解码。
```
Comment (ran-huang) [inline ticdc/ticdc-sink-to-kafka.md:316]:
```suggestion
开启了 `large-message-handle-compression` 之后，消费者收到的消息经过特定压缩协议编码，消费者应用程序需要使用指定的压缩协议进行数据解码。
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 5]
```diff
@@ -353,4 +374,76 @@ Kafka 消费者收到消息之后，首先检查 `onlyHandleKey` 字段。如果
 
 > **警告：**
 >
-> 在 Kafka 消费者处理数据并查询 TiDB 时，可能发生数据已经被 GC 的情况。你需要[调整 TiDB 集群的 GC Lifetime 设置](/system-variables.md#tidb_gc_life_time-从-v50-版本开始引入) 为一个较大的值，以避免该情况。
\ No newline at end of file
+> 在 Kafka 消费者处理数据并查询 TiDB 时，可能发生数据已经被 GC 的情况。你需要[调整 TiDB 集群的 GC Lifetime 设置](/system-variables.md#tidb_gc_life_time-从-v50-版本开始引入) 为一个较大的值，以避免该情况。
+
+### 发送大消息到外部存储
+
+从 v7.4.0 开始，TiCDC Kafka sink 支持在消息大小超过限制时将该条消息发送到外部存储服务，同时发送一条含有该条大消息在外部存储服务中的地址的消息到 Kafka。这样可以避免因为消息大小超过 Kafka Topic 限制而导致 changefeed 发生错误和同步任务失败的情况。
```
Comment (nongfushanquan) [inline ticdc/ticdc-sink-to-kafka.md:381]: 这样可以避免因为消息大小超过 Kafka Topic 限制而导致 changefeed 失败的情况。

Comment (ran-huang) [inline ticdc/ticdc-sink-to-kafka.md:381]:
```suggestion
从 v7.4.0 开始，TiCDC Kafka sink 支持在消息大小超过限制时将该条消息发送到外部存储服务，同时向 Kafka 发送一条含有该大消息在外部存储服务中的地址的消息。这样可以避免由于消息大小超过 Kafka Topic 限制而导致 changefeed 发生错误和同步任务失败的情况。
```

File: ticdc/ticdc-sink-to-kafka.md (hunk 4 above)
Comment (nongfushanquan) [inline ticdc/ticdc-sink-to-kafka.md:316]: 这个地方有简单示例吗

File: ticdc/ticdc-sink-to-kafka.md [hunk 6]
```diff
@@ -353,4 +374,76 @@ Kafka 消费者收到消息之后，首先检查 `onlyHandleKey` 字段。如果
...
+    ],
+    "old": null,
+    "_tidb": {     // TiDB 的扩展字段
+        "commitTs": 163963314122145239,
+        "claimCheckLocation": "s3:/claim-check-bucket/${uuid}.json"
+    }
+}
+```
+
+如果收到的消息有 `claimCheckLocation` 字段，根据该字段提供的地址，读取出以 JSON 格式存储的大消息数据。消息格式如下：
```
Comment (ran-huang) [inline ticdc/ticdc-sink-to-kafka.md:440]:
```suggestion
如果收到的消息有 `claimCheckLocation` 字段，Kafka consumer 根据该字段提供的地址，读取出以 JSON 格式存储的大消息数据。消息格式如下：
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 7]
```diff
@@ -353,4 +374,76 @@ Kafka 消费者收到消息之后，首先检查 `onlyHandleKey` 字段。如果
...
+
+```json
+{
+  key: "xxx",
+  value: "xxx",
+}
+```
+
+key 和 value 分别存有编码后的大消息，原本应该发送到 Kafka 消息中的对应字段。消费者可以通过解析这两部分的数据，还原大消息的内容。
```
Comment (ran-huang) [inline ticdc/ticdc-sink-to-kafka.md:449]:
```suggestion
key 和 value 分别存有编码后的大消息，该消息原本应该发送到 Kafka 消息中的对应字段。消费者可以通过解析这两部分的数据，还原大消息的内容。
```

File: ticdc/ticdc-sink-to-kafka.md (hunk 4 above)
Comment (3AceShowHand) [reply ticdc/ticdc-sink-to-kafka.md:316]: 目前没有，后续添加。

File: ticdc/ticdc-sink-to-kafka.md [hunk 8]
```diff
@@ -353,4 +374,74 @@ Kafka 消费者收到消息之后，首先检查 `onlyHandleKey` 字段。如果
...
+
+### 发送大消息到外部存储
+
+从 v7.4.0 开始，TiCDC Kafka sink 支持在消息大小超过限制时将该条消息发送到外部存储服务，同时发送一条含有该条大消息在外部存储服务中的地址的消息到 Kafka。这样可以避免因为消息大小超过 Kafka Topic 限制而导致 changefeed 失败的情况。
+
+配置样例如下所示：
+
+```toml
+[sink.kafka-config.large-message-handle]
+# 该参数从 v7.3.0 开始引入
```
Comment (ran-huang) [inline ticdc/ticdc-sink-to-kafka.md:387]: 这个参数是 7.3.0 引入，但发送大消息到外部存储的功能是 7.4.0 才有的吗？

Comment (3AceShowHand) [reply ticdc/ticdc-sink-to-kafka.md:387]: 是的，7.3.0 只有 handle-key-only 这个子功能。7.4.0 添加 claim-check 和 compression

Comment (ran-huang) [reply ticdc/ticdc-sink-to-kafka.md:387]: `large-message-handle-option` 现在可以设置的值就是 "handle-key-only" 或者 "claim-check" 这两种哇？可以写到注释里吗

**Review by nongfushanquan — APPROVED** [review]

File: ticdc/ticdc-sink-to-kafka.md [hunk 9]
```diff
@@ -353,4 +374,74 @@ Kafka 消费者收到消息之后，首先检查 `onlyHandleKey` 字段。如果
...
+
+> **建议：**
+> 
+> 目前支持的外部存储服务与 BR 相同。详细参数说明请参考 [BR 备份存储服务的 URI 格式](/br/backup-and-restore-storages.md#格式说明)。
+
+TiCDC 不会清理外部存储服务上的消息，数据消费者需要自行管理外部存储服务。
+
+### 消费外部存储中的大消息
+
+首先，Kafka consumer 会收到一条含有大消息在外部存储服务中的地址的消息，格式如下：
```
Comment (ran-huang) [inline ticdc/ticdc-sink-to-kafka.md:404]:
```suggestion
Kafka consumer 会收到一条含有大消息在外部存储服务中的地址的消息，格式如下：
```

**Review by ran-huang — APPROVED** [review]

File: ticdc/ticdc-sink-to-kafka.md [hunk 10]
```diff
@@ -292,6 +292,29 @@ SELECT COUNT(*) FROM INFORMATION_SCHEMA.TIKV_REGION_STATUS WHERE DB_NAME="databa
...
+# 该参数从 v7.4.0 开始引入
+# 默认为空，即不开启编码时的压缩功能
+# 可选值有 "none"、"lz4"、"snappy", 默认为 "none"
+large-message-handle-compression = "none"
+```
+
+该功能和 Kafka producer 的压缩功能不同：
+
+* `large-message-handle-compression` 中指定的压缩算法，它启用的是对单条 Kafka 消息进行压缩，并且压缩是在与消息大小限制参数比较之前进行。
+* 用户可以在 `sink-uri` 中配置压缩算法，它所启用的压缩功能应用在整个发送数据请求，其中包含多条 Kafka 消息，并且是在和消息大小限制参数比较之后进行的。
```
Comment (ran-huang) [inline ticdc/ticdc-sink-to-kafka.md:314]:
```suggestion
* 用户可以在 `sink-uri` 中配置压缩算法，它所启用的压缩功能应用在整个发送数据请求，其中包含多条 Kafka 消息，并且压缩是在和消息大小限制参数比较之后进行的。
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 11]
```diff
@@ -353,4 +374,74 @@ Kafka 消费者收到消息之后，首先检查 `onlyHandleKey` 字段。如果
...
+
+从 v7.4.0 开始，TiCDC Kafka sink 支持在消息大小超过限制时将该条消息发送到外部存储服务，同时向 Kafka 发送一条含有该大消息在外部存储服务中的地址的消息。这样可以避免因为消息大小超过 Kafka Topic 限制而导致 changefeed 失败的情况。
+
+配置样例如下所示：
+
+```toml
+[sink.kafka-config.large-message-handle]
+# 该参数从 v7.3.0 开始引入
+# 默认为空，即消息超过大小限制后，同步任务失败
+# 设置为 "handle-key-only" 时，如果消息超过大小，data 字段内容只发送 handle key；如果依旧超过大小，同步任务失败
```
Comment (ran-huang) [inline ticdc/ticdc-sink-to-kafka.md:389]:
```suggestion
# 设置为 "handle-key-only" 时，如果消息超过大小，data 字段内容只发送 handle key；如果依旧超过大小，同步任务失败
# 设置为 `claim-check` 时，将该条消息发送到外部存储服务
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 12]
```diff
@@ -353,4 +374,75 @@ Kafka 消费者收到消息之后，首先检查 `onlyHandleKey` 字段。如果
...
+# 设置为 `claim-check` 时，将该条消息发送到外部存储服务
```
Comment (ran-huang) [inline ticdc/ticdc-sink-to-kafka.md:390]:
```suggestion
# 设置为 `claim-check` 时，如果消息超过大小，将该条消息发送到外部存储服务
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 13]
```diff
@@ -292,6 +292,29 @@ SELECT COUNT(*) FROM INFORMATION_SCHEMA.TIKV_REGION_STATUS WHERE DB_NAME="databa
 
 Kafka Topic 对可以接收的消息大小有限制，该限制由 [`max.message.bytes`](https://kafka.apache.org/documentation/#topicconfigs_max.message.bytes) 参数控制。当 TiCDC Kafka sink 在发送数据时，如果发现数据大小超过了该限制，会导致 changefeed 报错，无法继续同步数据。为了解决这个问题，TiCDC 新增一个参数 `large-message-handle-option` 并提供如下解决方案。
 
+目前，如下功能支持 Canal-JSON 和 Open Protocol 两种编码协议。使用 Canal-JSON 协议时，你需要在 `sink-uri` 中指定 `enable-tidb-extension=true` 参数。
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:295]:
```suggestion
目前，如下功能支持 Canal-JSON 和 Open Protocol 两种编码协议。使用 Canal-JSON 协议时，你需要在 `sink-uri` 中设置 `enable-tidb-extension=true`。
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 14]
```diff
@@ -292,6 +292,29 @@ SELECT COUNT(*) FROM INFORMATION_SCHEMA.TIKV_REGION_STATUS WHERE DB_NAME="databa
...
+### TiCDC 层数据压缩功能
+
+从 v7.4.0 开始，TiCDC Kafka sink 支持在编码消息后立即对数据进行压缩，并与消息大小限制参数比较。该功能能够有效减少超过消息大小限制的情况发生。
+
+配置样例如下所示：
+
+```toml
+[sink.kafka-config.large-message-handle]
+# 该参数从 v7.4.0 开始引入
+# 默认为空，即不开启编码时的压缩功能
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:306]:
```suggestion
# 默认为 "none"，即不开启编码时的压缩功能
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 15]
```diff
@@ -353,4 +374,75 @@ Kafka 消费者收到消息之后，首先检查 `onlyHandleKey` 字段。如果
...
+从 v7.4.0 开始，TiCDC Kafka sink 支持在消息大小超过限制时将该条消息发送到外部存储服务，同时向 Kafka 发送一条含有该大消息在外部存储服务中的地址的消息。这样可以避免因为消息大小超过 Kafka Topic 限制而导致 changefeed 失败的情况。
+
+配置样例如下所示：
+
+```toml
+[sink.kafka-config.large-message-handle]
+# 该参数从 v7.3.0 开始引入
+# 默认为空，即消息超过大小限制后，同步任务失败
+# 设置为 "handle-key-only" 时，如果消息超过大小，data 字段内容只发送 handle key；如果依旧超过大小，同步任务失败
+# 设置为 `claim-check` 时，如果消息超过大小，将该条消息发送到外部存储服务
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:390]:
```suggestion
# 设置为 "claim-check" 时，如果消息超过大小，将该条消息发送到外部存储服务
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 16]
```diff
@@ -353,4 +374,75 @@ Kafka 消费者收到消息之后，首先检查 `onlyHandleKey` 字段。如果
...
+如果收到的消息有 `claimCheckLocation` 字段，Kafka consumer 根据该字段提供的地址，读取出以 JSON 格式存储的大消息数据。消息格式如下：
+
+```json
+{
+  key: "xxx",
+  value: "xxx",
+}
+```
+
+key 和 value 分别存有编码后的大消息，该消息原本应该发送到 Kafka 消息中的对应字段。消费者可以通过解析这两部分的数据，还原大消息的内容。
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:448]:
```suggestion
`key` 和 `value` 分别存有编码后的大消息，该消息原本应该发送到 Kafka 消息中的对应字段。消费者可以通过解析这两部分的数据，还原大消息的内容。
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 17]
```diff
@@ -353,4 +374,75 @@ Kafka 消费者收到消息之后，首先检查 `onlyHandleKey` 字段。如果
...
+
+> **建议：**
+> 
+> 目前支持的外部存储服务与 BR 相同。详细参数说明请参考 [BR 备份存储服务的 URI 格式](/br/backup-and-restore-storages.md#格式说明)。
+
+TiCDC 不会清理外部存储服务上的消息，数据消费者需要自行管理外部存储服务。
+
+### 消费外部存储中的大消息
+
+Kafka consumer 会收到一条含有大消息在外部存储服务中的地址的消息，格式如下：
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:405]:
```suggestion
Kafka 消费者会收到一条含有大消息在外部存储服务中的地址的消息，格式如下：
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 18]
```diff
@@ -353,4 +374,75 @@ Kafka 消费者收到消息之后，首先检查 `onlyHandleKey` 字段。如果
...
+    ],
+    "old": null,
+    "_tidb": {     // TiDB 的扩展字段
+        "commitTs": 163963314122145239,
+        "claimCheckLocation": "s3:/claim-check-bucket/${uuid}.json"
+    }
+}
+```
+
+如果收到的消息有 `claimCheckLocation` 字段，Kafka consumer 根据该字段提供的地址，读取出以 JSON 格式存储的大消息数据。消息格式如下：
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:439]:
```suggestion
如果收到的消息有 `claimCheckLocation` 字段，Kafka 消费者根据该字段提供的地址读取以 JSON 格式存储的大消息数据。消息格式如下：
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 19]
```diff
@@ -353,4 +374,75 @@ Kafka 消费者收到消息之后，首先检查 `onlyHandleKey` 字段。如果
...
+
+### 发送大消息到外部存储
+
+从 v7.4.0 开始，TiCDC Kafka sink 支持在消息大小超过限制时将该条消息发送到外部存储服务，同时向 Kafka 发送一条含有该大消息在外部存储服务中的地址的消息。这样可以避免因为消息大小超过 Kafka Topic 限制而导致 changefeed 失败的情况。
+
+配置样例如下所示：
+
+```toml
+[sink.kafka-config.large-message-handle]
+# 该参数从 v7.3.0 开始引入
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:387]:
```suggestion
# large-message-handle-option 从 v7.3.0 开始引入
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 20]
```diff
@@ -353,4 +374,75 @@ Kafka 消费者收到消息之后，首先检查 `onlyHandleKey` 字段。如果
...
+# 默认为空，即消息超过大小限制后，同步任务失败
+# 设置为 "handle-key-only" 时，如果消息超过大小，data 字段内容只发送 handle key；如果依旧超过大小，同步任务失败
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:389]:
```suggestion
# 设置为 "handle-key-only" 时，如果消息超过大小，data 字段内容只发送 Handle Key。如果依旧超过大小，同步任务失败
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 21]
```diff
@@ -353,4 +374,75 @@ Kafka 消费者收到消息之后，首先检查 `onlyHandleKey` 字段。如果
...
+# 设置为 `claim-check` 时，如果消息超过大小，将该条消息发送到外部存储服务
+large-message-handle-option = "claim-check"
+claim-check-storage-uri = "s3://claim-check-bucket"
+```
+
+当指定 `large-message-handle-option` 为 `claim-check` 时，`claim-check-storage-uri` 必须设置为一个有效的外部存储服务地址，否则创建 Changefeed 将会报错。
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:395]:
```suggestion
当指定 `large-message-handle-option` 为 `claim-check` 时，`claim-check-storage-uri` 必须设置为一个有效的外部存储服务地址，否则创建 changefeed 将会报错。
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 22]
```diff
@@ -292,6 +292,29 @@ SELECT COUNT(*) FROM INFORMATION_SCHEMA.TIKV_REGION_STATUS WHERE DB_NAME="databa
...
+
+从 v7.4.0 开始，TiCDC Kafka sink 支持在编码消息后立即对数据进行压缩，并与消息大小限制参数比较。该功能能够有效减少超过消息大小限制的情况发生。
+
+配置样例如下所示：
+
+```toml
+[sink.kafka-config.large-message-handle]
+# 该参数从 v7.4.0 开始引入
+# 默认为空，即不开启编码时的压缩功能
+# 可选值有 "none"、"lz4"、"snappy", 默认为 "none"
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:307]:
```suggestion
# 可选值有 "none"、"lz4"、"snappy"，默认为 "none"
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 23]
```diff
@@ -353,4 +374,75 @@ Kafka 消费者收到消息之后，首先检查 `onlyHandleKey` 字段。如果
 
 > **警告：**
 >
-> 在 Kafka 消费者处理数据并查询 TiDB 时，可能发生数据已经被 GC 的情况。你需要[调整 TiDB 集群的 GC Lifetime 设置](/system-variables.md#tidb_gc_life_time-从-v50-版本开始引入) 为一个较大的值，以避免该情况。
\ No newline at end of file
+> 在 Kafka 消费者处理数据并查询 TiDB 时，可能发生数据已经被 GC 的情况。你需要[调整 TiDB 集群的 GC Lifetime 设置](/system-variables.md#tidb_gc_life_time-从-v50-版本开始引入) 为一个较大的值，以避免该情况。
+
+### 发送大消息到外部存储
+
+从 v7.4.0 开始，TiCDC Kafka sink 支持在消息大小超过限制时将该条消息发送到外部存储服务，同时向 Kafka 发送一条含有该大消息在外部存储服务中的地址的消息。这样可以避免因为消息大小超过 Kafka Topic 限制而导致 changefeed 失败的情况。
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:381]:
```suggestion
从 v7.4.0 开始，TiCDC Kafka sink 支持在消息大小超过限制时将该条消息发送到外部存储服务，同时向 Kafka 发送一条含有该大消息在外部存储服务中地址的消息。这样可以避免因为消息大小超过 Kafka Topic 限制而导致 changefeed 失败的情况。
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 24]
```diff
@@ -353,4 +374,75 @@ Kafka 消费者收到消息之后，首先检查 `onlyHandleKey` 字段。如果
...
+
+配置样例如下所示：
+
+```toml
+[sink.kafka-config.large-message-handle]
+# 该参数从 v7.3.0 开始引入
+# 默认为空，即消息超过大小限制后，同步任务失败
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:388]:
```suggestion
# 默认为 "none"，即消息超过大小限制后，同步任务失败
```

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**